### PR TITLE
update to support apicurio for ocp 4.18

### PR DESCRIPTION
**Pull request by vi-patel**
update to support apicurio for ocp 4.18

### CODE REVIEW

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.18-lp-interop-service-registry-2-6-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.18-lp-interop-service-registry-2-6

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.18-lp-interop-service-registry-2-6

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.18-lp-interop-service-registry-2-6

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.18-lp-interop-service-registry-2-6-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.18-lp-interop-service-registry-2-6-fips

File: ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        {
+          "failure_rules":
+            [
+              {"step": "service-registry-run-tests", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "service-registry-run-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "APICURQE", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.18-lp","self-managed-lp","service-registry-lp"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
```
Comment (dfrazzette) [inline ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.18-lp-interop.yaml:66]:
```suggestion
      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
      FIREWATCH_DEFAULT_JIRA_ASSIGNEE: rkubis@redhat.com      
```

File: ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.18-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      FIREWATCH_DEFAULT_JIRA_ASSIGNEE: rkubis@redhat.com
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      USER_TAGS: |
+        scenario service-registry
+    test:
+    - ref: service-registry-run-tests
+    workflow: firewatch-ipi-aws
+  timeout: 4h0m0s
+- as: service-registry-2-6
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.18-lp-interop.yaml:52]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main-periodics.yaml [hunk 3]
```diff
@@ -646,3 +646,189 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build10
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main-periodics.yaml:651]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main-periodics.yaml [hunk 4]
```diff
@@ -646,3 +646,189 @@ periodics:
...
+        secretName: manifest-tool-local-pusher
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build10
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main-periodics.yaml:744]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.18-lp-interop.yaml (hunk 1 above)
Comment (vi-patel) [reply ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.18-lp-interop.yaml:68]: added a commit to add the ASSIGNEE

**Pull request merged**